doc/docusaurus/docs/delve-deeper/further-resources/academic-papers.md

```diff
@@ -1,28 +1,31 @@
 ---
-sidebar_position: 40
+sidebar_position: 30
 ---
 
-# Further Resources
+# Academic Papers
```

Comment:
"Academic Papers" is the same as "Research Publications". And "Video Resources" should not be under "Academic Papers".

Also, these are not necessarily all "academic", e.g., there's a technical report.

I suggest making a single list called "Papers and Videos"

Reply:
I ended up splitting "Academic Papers" into two pages:
- "Research publications"
- "Videos" (contains embedded youtube video)